internal/exec/stages/files/files.go

```diff
@@ -205,8 +216,14 @@ func (tmp linkEntry) create(l *log.Logger, u util.Util) error {
 	}
 
 	if err := l.LogOp(
-		func() error { return u.WriteLink(s) },
-		"writing link %q -> %q", s.Path, s.Target,
+		func() error {
+			err := u.DeletePathOnOverwrite(s.Node)
```

Comment:
nit: I like the `if err := func(); err != nil {`

but I don't actually care that much